scripts/src/java/org/oppia/android/scripts/maven/GenerateMavenDependenciesList.kt

```diff
@@ -175,6 +175,8 @@ class GenerateMavenDependenciesList(
     }
   }
 
+  // TODO(#3486): Update GenerateMavenDependenciesList.kt to call out first co-ordinate name
```

Comment:
Main nit: can you move this next to the code that actually prints out the text that we want to update? That seems a bit more natural of a location.

Reply:
Done. (But also kept here as it also involves some part)